docs/testplan/Convergence measurement in data center networks.md

```diff
@@ -0,0 +1,331 @@
+
+### Test Case # 2.9
+
+<p float="left">
+  <img src="Img/T2_Convergence_Downlink_Ungraceful_Restart.png" width="500"  hspace="150"/>
+</p>
+
+| **Test Name**  &nbsp;     | **Measure the convergence time during ungraceful restart of downlink linecard.**                         |
+| ------------------- | :--------------------------------------- |
+| **Topology**        | **Topology 2g**                          | 
+| **Type**            | **Scalability and Performance**          |
+| **Traffic**         | **Outbound**                             |
+| **Test Steps**      | 1. Send traffic from server port(tgen) connected to T1 to the eBGP prefixes.<br/>2. Traffic will be equally distributed between the available paths.<br/>3. Use command (nohup bash -c "sleep 5 && echo c > /proc/sysrq-trigger" &) to create panic on the downlink line card.<br/>4. Without TSA/TSB service, there might be traffic loss when the LC is coming up. It may advertise routes befor installing it in the ASIC.<br/>5. With TSA/TSB service, it should not see any loss.<br/>6. Measure the time it takes to converge the traffic to other available path without any loss.
+
+### Test Case # 2.10
+            
+<p float="left">
+  <img src="Img/T2_Convergence_Uplink_Ungraceful_Restart.png" width="500"  hspace="150"/>
+</p>
+
+| **Test Name** &nbsp;      | **Measure the convergence time during ungraceful restart of uplink linecard.**                         |
+| ------------------- | :--------------------------------------- |
+| **Topology**        | **Topology 2h**                          | 
+| **Type**            | **Scalability and Performance**          |
+| **Traffic**         | **Outbound**                             |
+| **Test Steps**      | 1. Send traffic from server port(tgen) connected to T1 to the eBGP prefixes.<br/>2. Traffic will be equally distributed between the available paths.<br/>3. Use command (nohup bash -c "sleep 5 && echo c > /proc/sysrq-trigger" &) to create panic on the uplink line card.<br/>4. Without TSA/TSB service, there might be traffic loss when the LC is going down till iBGP timer expiry and when the LC is coming up(It may advertise routes befor installing it in the ASIC).<br/>5. With TSA/TSB service, there might be traffic loss when the LC is going down till iBGP timer expiry and when LC is coming up, it should not see any loss.<br/>6. Measure the time it takes to converge the traffic to other available path without any loss.
+
+### Test Case # 2.11
+
+<p float="left">
+  <img src="Img/T2_Convergence_Sup_Ungraceful_Restart.png" width="500"  hspace="150"/>
+</p>
+
+| **Test Name** &nbsp;      | **Measure the convergence time during ungraceful restart of supervisor on T2 chassis.**                         |
```

Comment:
Please update the doc with details on how the convergence time will be calcuted for all these tests

Reply:
Added the measurement method under Test Methodology